Rubberduck.Parsing/VBA/IParseManager.cs

```diff
@@ -31,4 +31,55 @@ public interface IParseManager : IParserStatusProvider
         SuspensionResult OnSuspendParser(object requestor, IEnumerable<ParserState> allowedRunStates, Action busyAction, int millisecondsTimeout = -1);
         void MarkAsModified(QualifiedModuleName module);
     }
+
+
+
+    public enum SuspensionOutcome
```

Comment:
I wonder if we need a separate flag for where the suspension itself failed, as opposed to the underlying action failing?

Reply:
I'll add a separate flag for the failure around the read lock. I was honestly surprised to see that it used `UnexpectedError` although it is a known error condition. All other cases how the suspension itself can fail already have a separate outcome.